src/variableProcessor.ts

```diff
@@ -366,4 +398,70 @@ export class VariableProcessor {
 
         return variables;
     }
+
+    public static getRequestVariablesInCurrentFile(): Map<string, RequestVariableCacheValue> {
+        let editor = window.activeTextEditor;
+        if (!editor || !editor.document) {
+            return new Map<string, RequestVariableCacheValue>();
+        }
+
+        return VariableProcessor.getRequestVariablesInFile(editor.document);
+    }
+
+    public static getRequestVariablesInFile(document: TextDocument): Map<string, RequestVariableCacheValue> {
+        let variables = new Map<string, RequestVariableCacheValue>();
+
+        let text = document.getText();
+        let lines: string[] = text.split(/\r?\n/g);
+        let documentUri = document.uri.toString();
+        lines.forEach(line => {
+            let match: RegExpExecArray;
+            if (match = Constants.RequestVariableDefinitionRegex.exec(line)) {
+                let key = match[1];
+                const response = RequestVariableCache.get(new RequestVariableCacheKey(key, documentUri));
+                if (response) {
+                    variables.set(key, response);
+                }
+            }
+        });
+
+        return variables;
+    }
+
+    public static async checkVariableDefinitionExists(document: TextDocument, variableNames: string[]): Promise<{ name: string, exists: boolean }[]> {
+        let definitions = await VariableProcessor.getVariableDefinitionsInFile(document);
+
+        return variableNames.map((name) =>
+            ({ name, exists: definitions.has(name) })
+        );
+    }
+
+    public static async getVariableDefinitionsInFile(document: TextDocument): Promise<Map<string, VariableType[]>> {

```

Comment:
Make modifier to __private__

Reply:
Sure, was thinking this is a utility-method which can be used in the future. But it can be made public again then. I'll update 👍